src/style.css

```diff
@@ -0,0 +1,113 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  flex-grow: 1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+
+  height: 300px;
+
+  background-color: hsl(0, 100%, 50%);
+}
+
+.block_2 {
+  background-color: hsl(0, 100%, 40%);
+}
+
+.block_3 {
+  background-color: hsl(0, 100%, 30%);
+}
+
+.block_4 {
+  background-color: hsl(0, 100%, 20%);
+}
+
+.block_5 {
+  background-color: hsl(0, 100%, 10%);
+}
+
+.block_6 {
+  background-color: hsl(0, 100%, 0%);
+}
+
+@media (min-width: 300px) {
+  .container {
+    flex-direction: column;
+  }
```

Comment:
```suggestion
```
Just FYI - you may not change flex-direction at all (here and in other media queries) just by giving your `.block` property `flex-basis: 100%` (like you do in your `min-width: 900px` query)